src/PointwiseFunctions/GeneralRelativity/InterfaceNullNormal.hpp

```diff
@@ -0,0 +1,78 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
```

Comment:
Is this used?

Reply:
Nope, removed it. I was planning to add compute tags with these functions but realized that just functions might be enough for now.